api-report/driver-utils.api.md

```diff
@@ -113,7 +113,7 @@ export function combineAppAndProtocolSummary(appSummary: ISummaryTree, protocolS
 export function configurableUrlResolver(resolversList: IUrlResolver[], request: IRequest): Promise<IResolvedUrl | undefined>;
 
 // @public (undocumented)
-export function createGenericNetworkError(errorMessage: string, canRetry: boolean, retryAfterMs?: number, props?: ITelemetryProperties): GenericNetworkError | ThrottlingError;
```

Comment:
Not sure why local build did this...

Reply:
Ooh I saw that too on another branch.  Check it in please.  I wonder if I merged a PR without that change or something? Kind of suspicious but also pretty harmless if you or someone else fixes it up.